<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailWindow</class>
 <widget class="QWidget" name="DetailWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Detaylar</string>
  </property>
  <property name="windowIcon">
   <iconset resource="files.qrc">
    <normaloff>:/images/images/logo.ico</normaloff>:/images/images/logo.ico</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="leftLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="topLayout1">
       <item>
        <widget class="QLabel" name="code1_label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <italic>false</italic>
           <underline>true</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel
{
	color: rgb(245, 121, 0);
}</string>
         </property>
         <property name="text">
          <string>code1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="fpLayout1">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <property name="horizontalSpacing">
            <number>10</number>
           </property>
           <property name="verticalSpacing">
            <number>15</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Similarity:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Total fingerprints:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="code1_totalFP">
             <property name="styleSheet">
              <string notr="true">QLabel
{
	background: #FFF;
	border-radius: 1px solid black;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QProgressBar" name="simBar1">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="value">
              <number>50</number>
             </property>
             <property name="textVisible">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="spacing">
            <number>15</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Selected fingerprints:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="code1_selectedFP">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel
{
	background: #FFF;
	border-radius: 1px solid black;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Matched fingerprints:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="code1_matchedFP">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel
{
	background: #FFF;
	border-radius: 1px solid black;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="midLayout1">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="buttonLayout1">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QPushButton" name="code1_pushDoc">
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	color: #FFF;
	background: rgb(10, 54, 86)
}</string>
           </property>
           <property name="text">
            <string>Document</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="code1_pushSkeleton">
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	color: #FFF;
	background: rgb(10, 54, 86)
}</string>
           </property>
           <property name="text">
            <string>Skeleton</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="code1_stack">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="page">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>6</number>
           </property>
           <item row="0" column="0">
            <widget class="QTextBrowser" name="code1_docText">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QTextBrowser
{
	background: #FFF;
	padding: 5px;
}</string>
             </property>
             <property name="documentTitle">
              <string/>
             </property>
             <property name="lineWrapMode">
              <enum>QTextEdit::WidgetWidth</enum>
             </property>
             <property name="lineWrapColumnOrWidth">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QTextBrowser" name="code1_skeletonText">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QTextBrowser
{
	background: #FFF;
	padding: 5px;
}</string>
             </property>
             <property name="documentTitle">
              <string/>
             </property>
             <property name="lineWrapMode">
              <enum>QTextEdit::WidgetWidth</enum>
             </property>
             <property name="lineWrapColumnOrWidth">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="code1_pushOpen">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	color: #FFF;
	background: rgb(245, 121, 0)
}</string>
       </property>
       <property name="text">
        <string>Click here to open the file in default program.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="rightLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="topLayout2">
       <item>
        <widget class="QLabel" name="code2_label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <italic>false</italic>
           <underline>true</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel
{
	color: rgb(245, 121, 0);
}</string>
         </property>
         <property name="text">
          <string>code2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="fpLayout2">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="horizontalSpacing">
            <number>10</number>
           </property>
           <property name="verticalSpacing">
            <number>15</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Similarity:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QProgressBar" name="simBar2">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="value">
              <number>50</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Total fingerprints:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="code2_totalFP">
             <property name="styleSheet">
              <string notr="true">QLabel
{
	background: #FFF;
	border-radius: 1px solid black;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_7">
           <property name="spacing">
            <number>15</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Selected fingerprints:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="code2_selectedFP">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel
{
	background: #FFF;
	border-radius: 1px solid black;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Matched fingerprints:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="code2_matchedFP">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel
{
	background: #FFF;
	border-radius: 1px solid black;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="midLayout2">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="buttonLayout2">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QPushButton" name="code2_pushDoc">
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	color: #FFF;
	background: rgb(10, 54, 86)
}</string>
           </property>
           <property name="text">
            <string>Document</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="code2_pushSkeleton">
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	color: #FFF;
	background: rgb(10, 54, 86)
}</string>
           </property>
           <property name="text">
            <string>Skeleton</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="code2_stack">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="page_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>6</number>
           </property>
           <item row="0" column="0">
            <widget class="QTextBrowser" name="code2_docText">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QTextBrowser
{
	background: #FFF;
	padding: 5px;
}</string>
             </property>
             <property name="documentTitle">
              <string/>
             </property>
             <property name="lineWrapMode">
              <enum>QTextEdit::WidgetWidth</enum>
             </property>
             <property name="lineWrapColumnOrWidth">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QTextBrowser" name="code2_skeletonText">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QTextBrowser
{
	background: #FFF;
	padding: 5px;
}</string>
             </property>
             <property name="documentTitle">
              <string/>
             </property>
             <property name="lineWrapMode">
              <enum>QTextEdit::WidgetWidth</enum>
             </property>
             <property name="lineWrapColumnOrWidth">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="code2_pushOpen">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	color: #FFF;
	background: rgb(245, 121, 0)
}</string>
       </property>
       <property name="text">
        <string>Click here to open the file in default program.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="files.qrc"/>
 </resources>
 <connections/>
</ui>
